python/snewpy/_model_downloader.py

```diff
@@ -100,8 +100,7 @@ def load(self) -> Path:
             logger.info(f'Downloading file {self.path}')
             _download(self.remote, self.path)
             self.check()
-        finally:
-            return self.path
```

Comment:
Just a note: This change is not strictly necessary to fix the integration test, but it was something we discovered while debugging—if the return statement is in a `finally` block, any errors occuring in the `except` block (e.g. during the download) are suppressed.